### PR TITLE
Feature/4 login

### LINKED ISSUE #4: Create empty login page
Create a base login page

Include
- [ ] Email
- [ ]  Password
- [ ]  Submit/Login button
- [ ]  Sign-up button

Check out the [Figma example](https://www.figma.com/file/b1HcXwM8aQO8PHK31YXwuK/Figma-Practice?type=design&node-id=23-81&mode=design&t=QjvXUHdRqT8UIvgE-0)

### PR DESCRIPTION
**Pull request by GCherry**
Added a skeleton template for tab and stack routing

### CODE REVIEW

File: frontApp/app/(tabs)/friends.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { View, Text, Touchable, TouchableOpacity, StyleSheet, Image, ScrollView } from 'react-native';
+import React, { useEffect, useState } from 'react';
+import { Link } from 'expo-router';
+import { Ionicons } from '@expo/vector-icons'
```
Comment (jameslac) [inline frontApp/app/(tabs)/friends.tsx]: missing a semicolon at the end to keep consistent with all other imports

File: frontApp/app/(tabs)/_layout.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { View, Text } from 'react-native';
+import React from 'react';
+import { Stack, Tabs } from 'expo-router';
+import { Ionicons } from '@expo/vector-icons'
```
Comment (jameslac) [inline frontApp/app/(tabs)/_layout.tsx:4]: missing a semicolon at the end to keep consistent with all other imports

File: frontApp/app/(tabs)/home.tsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import { View, Text, Touchable, TouchableOpacity, StyleSheet, Image, ScrollView } from 'react-native';
+import React, { useEffect, useState } from 'react';
+import { Link } from 'expo-router';
+import { Ionicons } from '@expo/vector-icons'
```
Comment (jameslac) [inline frontApp/app/(tabs)/home.tsx]: missing a semicolon at the end to keep consistent with all other imports

File: frontApp/app/index.tsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import { View, Text, Touchable, TouchableOpacity, StyleSheet, Image, ScrollView, Button } from 'react-native';
+import React, { useEffect, useState } from 'react';
+import { Link, useRouter } from 'expo-router';
+import { Ionicons } from '@expo/vector-icons'
```
Comment (jameslac) [inline frontApp/app/index.tsx]: missing a semicolon at the end to keep consistent with all other imports

File: frontApp/app/register.tsx [hunk 5]
```diff
@@ -0,0 +1,14 @@
+import { View, Text, Touchable, TouchableOpacity, StyleSheet, Image, ScrollView } from 'react-native';
+import React, { useEffect, useState } from 'react';
+import { Link } from 'expo-router';
+import { Ionicons } from '@expo/vector-icons'
```
Comment (jameslac) [inline frontApp/app/register.tsx]: missing a semicolon at the end to keep consistent with all other imports

File: frontApp/babel.config.js [hunk 6]
```diff
@@ -1,6 +1,7 @@
-module.exports = function(api) {
+module.exports = function (api) {
   api.cache(true);
   return {
     presets: ['babel-preset-expo'],
+    plugins: ['expo-router/babel', 'react-native-reanimated/plugin'],
```
Comment (jameslac) [inline frontApp/babel.config.js]: does the comma at the end of this line not break the return statement?

Comment (GCherry) [reply frontApp/babel.config.js]: It doesn't break it but I'll remove it

**Review by jameslac — APPROVED** [review]

**Review by JCherry022 — APPROVED** [review]

**Pull request merged**